tools/project_manager/show_word_context.php

```diff
@@ -81,6 +81,20 @@
     echo "</a>";
     echo "</p>";
 
+    echo "<form method='GET' id='wordInstancesForm'>";
+    echo "<input type='hidden' name='projectid' value='$projectid' />";
+    echo "<input type='hidden' name='word' value='$encWord' />";
+    echo "<input type='hidden' name='layout' value='$layout' />";
+    echo "<input type='hidden' name='frame' value='left' />";
+    echo "<label for='wordInstancesSelect'>" . _("Number of word context results: ") . "</label>";
+    echo "<select id='wordInstancesSelect' name='wordInstances' style='margin-left: 2px;' onchange='$(\"#wordInstancesForm\").submit()'>";
+    foreach(range(10, 100, 10) as $option) {
+        echo "<option value='$option'" . ($option == $wordInstances ? "selected" : "") . ">$option</option>";
```

Comment:
Add a space after `'$option'` so there's one before `selected`.

Reply:
I put it before the word selected later on so it is only present if we are on the selected one. Not that it matters much.